plans/main.fmf

```diff
@@ -2,8 +2,24 @@ execute:
     how: tmt
 discover:
     how: fmf
-environment:
+
+environment+:
     NO_COLOR: 1
+
+adjust+:
+    - environment+:
+        SYSTEM_RELEASE: centos-7
+      when: distro == centos-7
+    - environment+:
+        SYSTEM_RELEASE: oracle-7
+      when: distro == oraclelinux-7
+    - environment+:
+        SYSTEM_RELEASE: centos-8
+      when: distro == centos-8-latest
+    - environment+:
+        SYSTEM_RELEASE: oracle-8
+      when: distro == oraclelinux-8
```

Comment:
We probably need to adjust this to include 8.4 and 8.6

```suggestion
adjust+:
    - environment+:
        SYSTEM_RELEASE: centos-7
      when: distro == centos-7
    - environment+:
        SYSTEM_RELEASE: oracle-7
      when: distro == oraclelinux-7
    - environment+:
        SYSTEM_RELEASE: centos-8.4
      when: distro == centos-8.4
    - environment+:
        SYSTEM_RELEASE: centos-8
      when: distro == centos-8-latest
    - environment+:
        SYSTEM_RELEASE: oracle-8.4
      when: distro == oraclelinux-8.4
    - environment+:
        SYSTEM_RELEASE: oracle-8.6
      when: distro == oraclelinux-8.6
```

Reply:
Good catch! Thanks

Reply:
Also, updated the `centos-8` to become `centos-8.5`.

Reply:
(maybe it's not necessary)

Reply:
You're right, it would be better to differentiate